2-functions/2-arguments/1-before.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+final class ReportGenerator
+{
+
+    // ...
+
+    public function generate($userId, $firstName, $lastName, $startDate, $endEnd, $salesReport)
```

Comment:
better show them the doc block as well so they see that `$salesReport` is a bool we can focus on it